bindings/rust/evmc-vm/src/container.rs

```diff
@@ -61,6 +62,20 @@ mod tests {
         }
     }
 
+    unsafe extern "C" fn get_dummy_tx_context(
```

Comment:
This seems to be a duplicate of the one in lib.rs. Perhaps we could update the one in lib.rs and include it?

Reply:
It is not an exact copy though.